examples/rh-sso/rh-sso.conf

```diff
@@ -0,0 +1,33 @@
+server {
+    listen 8180;
+    resolver 8.8.8.8;
+
+    set_by_lua $rhsso_endpoint 'return os.getenv("RHSSO_ENDPOINT")';
+
+    location = /register_client {
+      internal;
+
+      proxy_pass $registration_url;
+
+      proxy_set_header Content-Type "application/json";
+      proxy_set_header Accept "application/json";
+      proxy_set_header Authorization "Bearer $access_token";
+    }
+
+    # Proxying RH-SSO resources (stylesheets etc.)
+    location ~ ^/auth/resources {
+      proxy_pass $rhsso_endpoint;
+    }
+    location ~ ^/favicon.ico {
+      proxy_pass $rhsso_endpoint;
+    }
+
+    # URL that will listen to the Webhooks
+    location ~ ^/webhooks/?$ {
+      set $client_id null;
+      set $access_token null;
+      set $registration_url null;
+      access_by_lua "require('client-registrations/webhook_handler').handle()";
```

Comment:
Shouldn't be `content_by_lua` more meaningful ?  This is not access policy.

Also by using `content_by_lua_block` you can embed it inside and have just one file.